data/json/npcs/tacoma_ranch/NPC_ranch_foreman.json

```diff
@@ -367,7 +367,7 @@
   {
     "id": "MISSION_RANCH_FOREMAN_6",
     "type": "mission_definition",
-    "name": { "str": "Gather 75 Rocks" },
+    "name": { "str": "Gather 75 rocks" },
```

Comment:
Why lowercase for this specific mission but not other missions in the same file?

Reply:
I focused on the Isherwoods and did not check that entire file. Fixed.

Reply:
However the convention of mission names seems to be capitalising initial letters of all words.